uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/my-leave/components/my-leave-report_test.js

```diff
@@ -641,8 +641,8 @@
             actionMatrix = getActionMatrixForStatus('cancelled');
           });
 
-          it('shows no actions', function () {
-            expect(actionMatrix.length).toBe(0);
+          it('shows the "view" action', function () {
+            expect(actionMatrix.length).toBe(1);
```

Comment:
Instead of checking length, we should check the content
`expect(actionMatrix.length).toEqual(['view']);`
Because current check is true when array has some other value than `view`, which should not be the case.